src/plugins/discover/public/application/components/sidebar/discover_sidebar.scss

```diff
@@ -72,28 +64,28 @@
 
 .dscSidebarItem--active {
   border-top: 1px solid $euiColorLightShade;
-  background: shade($euiColorLightestShade, 5%);
   color: $euiColorFullShade;
   .euiText {
     font-weight: bold;
   }
 }
 
 .dscSidebarField {
-  padding: $euiSizeXS 0;
+  padding: $euiSizeXS;
   display: flex;
-  align-items: flex-start;
+  align-items: center;
   max-width: 100%;
-  margin: 0;
   width: 100%;
   border: none;
-  border-radius: 0;
+  border-radius: $euiBorderRadius - 1px;
   text-align: left;
 }
 
 .dscSidebarField__name {
   margin-left: $euiSizeS;
   flex-grow: 1;
+  word-break: break-word;
+  padding-right: $euiSizeXS * 0.25;
```

Comment:
1px padding?

Reply:
That is to avoid having the field name too close to the button. This as an example, no padding and 1px padding

<img width="250" alt="Group 3" src="https://user-images.githubusercontent.com/4016496/87462142-50517900-c5c4-11ea-8a50-24507bcc5753.png">

Reply:
👍 Gotcha. Since the math just ends up being `1px` your fine to just say `1px` it's anything above that that requires math on our sizing variables